testData/highlighting/unitializedStructInitialization.go

```diff
@@ -0,0 +1,30 @@
+package demo2094
+
+type demo struct {
+	a string
+	b int32
+	c string
+}
+
+type bemo struct {
+	demo
+	x string
+}
+
+func _() (struct {
+	x string
+}) {
+	_ = struct{ x string }{x: "demo"}
+	_ = <weak_warning descr="Structure has unnamed field initialization">demo{"demo"}</weak_warning>
+	b, _ := <weak_warning descr="Structure has unnamed field initialization">demo{"demo"}</weak_warning>, 1
+	_ = <weak_warning descr="Structure has unnamed field initialization">demo{"demo", 1, "demo"}</weak_warning>
+	_ = demo{a: "demo"}
+	_ = demo{a: "demo", b: 1}
+	_ = <weak_warning descr="Structure has unnamed field initialization">demo{
```

Comment:
Sorry, I've missed that you didn't change reporting element. As @sjamesr said, it's good to keep inspection in consistent state, so if `missing key` error is reported on GoElement then this inspection also should be reported on GoElement too, not on whole literal